HW04/Anastasiya_Gavrilik/spec/custom/my_each_spec.rb

```diff
@@ -0,0 +1,27 @@
+RSpec.describe Custom do
+  describe 'my_each' do
```

Comment:
http://www.betterspecs.org/#describe
```suggestion
  describe '#my_each' do
```

Reply:
Done, thanks